src/components/AddCommentForm.jsx

```diff
@@ -0,0 +1,61 @@
+import { useState } from "react";
+import { postCommentByArticleId } from "../api/api";
+
+function AddCommentForm({ articleId, comments, setComments, setErr }) {
+  const [comment, setComment] = useState("");
+  const [btnDisabled, setBtnDisabled] = useState(false);
+
+  function handleSubmit(articleId, comment) {
+    if (comment.length > 1) {
+      setComments([
+        ...comments,
+        {
+          comment_id: "newComment",
+          article_id: articleId,
+          author: "jessjelly",
+          body: comment,
+          created_at: new Date().toString(),
+          votes: 0,
+        },
+      ]);
+      setBtnDisabled(true);
+
+      setTimeout(() => {
+        postCommentByArticleId(articleId, comment)
+          .then((newComment) => {
+            setComments([...comments, newComment[0]]);
+            setComment("");
+            setBtnDisabled(false);
+          })
+          .catch((err) => {
+            console.log(comments);
+
+            setComments([...comments]);
+            setErr("Something went wrong");
+            setComment("");
+          });
+      }, 5000);
```

Comment:
Im not sure why this is here, and why 5 seconds was chosen. Shouldnt rely on setTimeout for async stuff, wont always be quick.